cinch/roles/jenkins_master/templates/roles_and_ldap_auth.groovy

```diff
@@ -0,0 +1,177 @@
+import hudson.*;
```

Comment:
Seems like Groovy may need its own style guide (yet to be defined), so let's just leave this as-is for now.